tools/generate-component/boilerplate/component.ts

```diff
@@ -36,13 +34,6 @@ export class __nameUpperCase__ extends LitElement {
     this._myEvent.emit();
   }
 
-  public override connectedCallback(): void {
-    super.connectedCallback();
-    const signal = this._abort.signal;
-    this.addEventListener('click', () => this._onClickFn(), { signal });
```

Comment:
I suggest to add the click event listener also here to the constructor. We still have the _onClickFn() function which is unused now.

Reply:
I think I would rather remove it from the template entirely (and maybe more). We have enough examples which can be used as reference.
What do you think?

Reply:
ok as well. boilerplate usage is rare anyways :)